1_fetch/out/subset_sites.yml

```diff
@@ -0,0 +1,2 @@
+site_id:
+  - '07374000'
```

Comment:
we can wait on this one until we know how we're handling sites, but the site list should also go into the shared viz_config.yml eventually

Reply:
^ that seems like it would be difficult to do since the `viz_config` is editable by hand, but the site list will be auto-generated. Is it a good idea to mix that within the same file?

Reply:
i was under the impression that we had hand-picked final sites for at least one vizzy. is that not true?

Reply:
Hmm I was under the impression we were auto-generating this list as much as possible. Let's bring this over to slack